docs/ops/arithmetic/Atanh_3.md

```diff
@@ -4,34 +4,28 @@
 
 **Category**: Arithmetic unary operation
 
-**Short description**: *Atanh* performs element-wise hyperbolic inverse tangent (arctangenth) operation with given tensor.
+**Short description**: *Atanh* performs element-wise hyperbolic inverse tangent (arctangenth) operation with a given tensor.
 
-**Attributes**:
+**Detailed description**: *Atanh* performs element-wise hyperbolic inverse tangent (arctangenth) operation on a given input tensor, based on the following mathematical formula:
 
-    No attributes available.
+\f[ a_{i} = atanh(a_{i}) \f]
+
+**Attributes**: Atanh operation has no attributes.
 
 **Inputs**
 
 * **1**: A tensor of type *T*. **Required.**
 
 **Outputs**
 
-* **1**: The result of element-wise atanh operation. A tensor of type *T*.
+* **1**: The result of element-wise atanh operation applied to the input tensor. A tensor of type *T* and the same shape as input tensor.
 
 **Types**
 
-* *T*: any floating point type.
-
-*Atanh* does the following with the input tensor *a*:
-
-\f[
-a_{i} = atanh(a_{i})
-\f]
+* *T*: any supported numeric type.
```

Comment:
Do we support integer input type?
I see that fws supports only floating-point types

Reply:
hi @rkazants , I checked source code again: https://github.com/openvinotoolkit/openvino/blob/master/ngraph/core/src/op/atanh.cpp#L76
I found we only support float, so I will recall this description update.

Reply:
Now I see contractionary between the specification and ngraph implementation. In ngraph implementation it reports integer type as supported case.

Reply:
In general case we should think about the backward compatibility, and if operation had a support of integer types we need to extend the spec. @lazarevevgeny @rkazants Do you agree with me?

Reply:
I think same as [acosh](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/arithmetic/Acosh_3.md) and [asinh](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/arithmetic/Asinh_3.md), Atanh should also support integer types. To avoid backward incompatible changes, we could extend types in spec to any numeric type. In reference implementation, to overload atanh function in the same way as in [here ](https://github.com/openvinotoolkit/openvino/blob/master/ngraph/core/reference/include/ngraph/runtime/reference/acosh.hpp#L26-L34). Lastly, add SLTs for floating-point types [here](https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/activation.cpp#L29) (which is missing) and integer types [here](https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/activation.cpp#L68).

Reply:
Thanks @rkazants @ilyachur @ggalieroc, I had reverted my last commit, will support integer.

Reply:
@ilyachur, I agree. Thanks!